images/html-editor/insert-emoticon.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```

Comment:
Can you remove the `class="st0"` from the elements? Not needed.

Reply:
Sure, can do!

Reply:
Ah, whoops, those also show in the HTML file. I'll also remove them there.